Doc/library/errno.rst

```diff
@@ -29,16 +29,25 @@ defined by the module.  The specific list of defined symbols is available as
 
    Operation not permitted
 
+   .. seealso::
```

Comment:
The seealso:: notation adds a yellow box as a highlight, which is maybe a little too prominent -- I wondered if the links should just be part of the tiny paragraph.  ex. ("Operation not permitted.  This error is mapped to ...").  But 'seealso' is certainly the right logical concept to use, and EINTR/InterruptedError already set the pattern, so I think this approach is correct.

Reply:
I followed the pattern set by EINTR/InterruptedError, but I agree it may be a bit much, visually. Maybe it'd be better if we did as in the InterruptedError description in exceptions.rst, and just add the observation to the description instead of leaving it as a 'See Also' extra.
So we have the following in exceptions.rst:
```
.. exception:: InterruptedError

   Raised when a system call is interrupted by an incoming signal.
   Corresponds to :c:data:`errno` :py:data:`~errno.EINTR`.
```
And we could do the same in errno.rst:
```
.. data:: EINTR

   Interrupted system call.
   This error is mapped to the exception :exc:`InterruptedError`.
```

Reply:
I went ahead and tried it in just paragraph form, and I think it looks fine.  Because I had to do the necessary editing for the experiment anyway, I've gone ahead and pushed the commit to your branch -- no point making you re-do the same work! What do you think?

Reply:
I think it's great, really, thanks a lot for this!

Reply:
@yyyyyyyan: thanks! because you're a first-time contributor, you can also add yourself to the Misc/ACKS file if you like. Once you've done that, I'll finally merge this.

Reply:
Oh, that's great, I sure want it! Just committed the addition to Misc/ACKS :-)